src/containers/forms/OrganizationForm/index.tsx

```diff
@@ -0,0 +1,191 @@
+import { ErrorMessage, Field, Form, Formik } from 'formik';
+import moment from 'moment';
+import React, { useState } from 'react';
+import { Redirect } from 'react-router';
+import { Button, Label } from 'reactstrap';
+import { FormGroup } from 'reactstrap';
+import * as Yup from 'yup';
+import {
+  OPENSRP_ORGANIZATION_ENDPOINT,
+  ORGANIZATION_LABEL,
+  REQUIRED,
+  SAVE,
+  SAVING,
+  TEAM_LIST_URL,
+} from '../../../constants';
+import { generateNameSpacedUUID } from '../../../helpers/utils';
+import { OpenSRPService } from '../../../services/opensrp';
+
+const OrgFormNameSpace = '9a4c8cb0-df70-11e9-b38b-57f114a50538';
+
+/** default type value for organizations */
+const defaultOrganizationType = {
+  coding: [
+    {
+      code: 'team',
+      display: 'Team',
+      system: 'http://terminology.hl7.org/CodeSystem/organization-type',
+    },
+  ],
+};
+
+/** yup validation schema for teams Form input */
+export const OrgSchema = Yup.object().shape({
+  active: Yup.boolean(),
+  identifier: Yup.string(),
+  name: Yup.string().required(REQUIRED),
+});
+
+/** interface for data fields for team's form */
+interface OrganizationFormFields {
+  active: boolean;
+  identifier: string;
+  name: string;
+  type?: any;
```

Comment:
It seems like we know what kind of object goes here, no?  If so let us not use `any` as the type

Reply:
will cahnge this